crates/infra/cli/src/commands/run/mod.rs

```diff
@@ -19,26 +19,42 @@ enum RunCommand {
      * User-facing:
      *
      */
-    /// Run the public 'slang_solidity' crate shipped to Cargo users.
+    /// Runs the public 'slang_solidity' crate shipped to Cargo users.
     #[clap(name = "slang_solidity")]
     SlangSolidity,
-
-    /// Run smoke tests of the Solidity parser against source files from the Sanctuary repositories.
-    #[clap(name = "solidity_testing_smoke")]
-    SolidityTestingSmoke,
+    /// Runs the Solidity parser against source files from the Sanctuary repositories.
+    #[clap(name = "solidity_testing_sanctuary")]
+    SolidityTestingSanctuary,
 
     /*
      *
      * Hidden (for automation only):
      *
      */
+    /// Runs codegen for the Rust parser crate.
     #[clap(name = "solidity_cargo_build", hide = true)]
     SolidityCargoBuild,
-
+    /// Runs codegen for the NAPI parser crate.
     #[clap(name = "solidity_npm_build", hide = true)]
     SolidityNpmBuild,
 }
 
+impl RunCommand {
+    fn should_run_in_release_mode(&self) -> bool {
+        match self {
```

Comment:
optional style nit - I find that 'everything is an expression' in Rust helps simplify the code in many places, this could just be

```rust
match self {
   // This crate ...
   Self::SolidityTestingSanctuary => true,
   // These ...
   ... | ... => false,
}
```

Reply:
I don't really like that `return` is optional in Rust. It is an important keyword in almost all programming languages that describe exactly whether the expression on the right hand side results in a value or `()`/`void`. It is very useful when reading statements left to right to understand the context as the reader's eye is moving.

It is less of an issue in a tiny function like this, but in more complex ones, it is much more important, especially with multiple match cases or if branches.

If it is possible, I would have hoped to enforce it everywhere. Saving the few extra keystrokes is just not worth it.

Reply:
I am going to 100% disagree here. Rust is an expression-oriented language, so eliding the `return` *statement* is quite natural. I prefer it unless there are early returns, which I don't like in any case.